lib/Optimizer/Lower.cpp

```diff
@@ -192,6 +191,30 @@ static void lowerReluNode(Function *F, const ReluNode &R) {
   R.getResult().replaceAllUsesOfWith(relu);
 }
 
+static void lowerPadNode(Function *F, const PadNode &P) {
```

Comment:
Can you please assert that the Pad node is in the right mode (const)?

Reply:
Done.